package.json

```diff
@@ -39,7 +39,7 @@
     "electron-builder": "5.2.1",
     "electron-connect": "^0.3.7",
     "electron-packager": "^7.0.1",
-    "electron-prebuilt": "1.2.6",
+    "electron-prebuilt": "1.3.1",
```

Comment:
Please keep v1.2.6 here. v1.3 should be tested in #214.
Please DON'T multiple things in one PR...